mongodbatlas/organizations.go

```diff
@@ -21,18 +21,24 @@ import (
 )
 
 const (
-	orgsBasePath = "api/atlas/v1.0/orgs"
+	orgsBasePath       = "api/atlas/v1.0/orgs"
+	invitationBasePath = orgsBasePath + "/%s/invites"
 )
 
 // OrganizationsService provides access to the organization related functions in the Atlas API.
 //
 // See more: https://docs.atlas.mongodb.com/reference/api/organizations/
 type OrganizationsService interface {
 	List(context.Context, *OrganizationsListOptions) (*Organizations, *Response, error)
+	ListUnacceptedInvitations(context.Context, string, *InvitationOptions) (*[]Invitation, *Response, error)
 	Get(context.Context, string) (*Organization, *Response, error)
+	GetUnacceptedInvitation(context.Context, string, string) (*Invitation, *Response, error)
```

Comment:
[nit] similar to above on naming we can make this 
```suggestion
	Invitation(context.Context, string, string) (*Invitation, *Response, error)
```

Usually the plural vs singular is enough and by the docs we don't mention unaccepted on the titles